File: code/1010/1010.rb
line1 = gets.split(' ')
line2 = gets.split(' ')

quant1 = line1[1].to_i
value1 = line1[2].to_f
quant2 = line2[1].to_i
value2 = line2[2].to_f

puts format('VALOR A PAGAR: R$ %.2f', quant1 * value1 + quant2 * value2)

File: code/1010/1010.c
#include <stdio.h>

int main() {

  int quant1, quant2;
  double value1, value2;

  scanf("%d %d %lf %d %d %lf", &quant1, &quant1, &value1, &quant2, &quant2,
        &value2);

  printf("VALOR A PAGAR: R$ %.2lf\n", quant1 * value1 + quant2 * value2);

  return 0;
}
